File: testdata/jennies/builders/map_of_disjunctions/JavaBuilder/map_of_disjunctions/PanelBuilder.java
package map_of_disjunctions;


public class PanelBuilder implements cog.Builder<Panel> {
    protected final Panel internal;
    
    public PanelBuilder() {
        this.internal = new Panel();
        this.internal.kind = "Panel";
    }
    public PanelBuilder title(String title) {
        this.internal.title = title;
        return this;
    }
    public Panel build() {
        return this.internal;
    }
}

File: testdata/jennies/builders/map_of_disjunctions/PHPBuilder/src/MapOfDisjunctions/PanelBuilder.php
<?php

namespace Grafana\Foundation\MapOfDisjunctions;

/**
 * @implements \Grafana\Foundation\Cog\Builder<\Grafana\Foundation\MapOfDisjunctions\Panel>
 */
class PanelBuilder implements \Grafana\Foundation\Cog\Builder
{
    protected \Grafana\Foundation\MapOfDisjunctions\Panel $internal;

    public function __construct()
    {
    	$this->internal = new \Grafana\Foundation\MapOfDisjunctions\Panel();
    $this->internal->kind = "Panel";
    }

    /**
     * Builds the object.
     * @return \Grafana\Foundation\MapOfDisjunctions\Panel
     */
    public function build()
    {
        return $this->internal;
    }

    public function title(string $title): static
    {
        $this->internal->title = $title;
    
        return $this;
    }

}
